src/components/Visualizations/inflictorWithValue.jsx

```diff
@@ -230,6 +238,16 @@ class InflictorWithValue extends React.Component {
             <div className="buffOverlay">
               {value > 0 && value}
             </div>
+          }
+            {type === 'purchase' &&
```

Comment:
Don't think we need to check for type here if we aren't providing charges anywhere else